transform/macros/tooling/create_snowflake_databases.sql

```diff
@@ -0,0 +1,19 @@
+{# Macro for creating a list of Databases in Snowflake. #}
+{# 
+    Run using
+    dbt run-operation create_snowflake_databases  --args 'objects_to_be_created'
+#}
+
+{% macro create_snowflake_databases(objects_to_be_created) %}
+    {% for database in objects_to_be_created %}        
+        
+            {% set create_database_sql %}
+                use role transformer_dbt_prd;
```

Comment:
oh, I think I see why you have that above, I dont think we want to do it this way, we want sysadmin to create the database, the user that runs this will need to have that role otherwise the script will fail which is what we want